third_party/packages/ansi_up/lib/ansi_up.dart

```diff
@@ -304,11 +304,11 @@ enum PacketKind {
 }
 
 class _TextPacket {
-  _TextPacket({required this.kind, this.text = '', this.url = ''});
+  _TextPacket({required this.kind});
 
   PacketKind kind;
-  String text;
-  final String url;
+  String text = '';
+  final String url = '';
```

Comment:
why is this change necessary?

Reply:
I got warning from analyzer, that the parameter is never used, that is true.